apps/cowswap-frontend/src/modules/tokensList/utils/tokensListSorter.ts

```diff
@@ -0,0 +1,20 @@
+import { TokenWithLogo } from '@cowprotocol/common-const'
+import { getIsNativeToken } from '@cowprotocol/common-utils'
+
+import { TokenAmounts } from 'modules/tokens'
+
+export function tokensListSorter(balances: TokenAmounts): (a: TokenWithLogo, b: TokenWithLogo) => number {
+  return (a: TokenWithLogo, b: TokenWithLogo) => {
+    const aBalance = balances[a.address]
+    const bBalance = balances[b.address]
+
+    // Native always first
+    if (getIsNativeToken(a) || getIsNativeToken(b)) return 1
```

Comment:
Isn't this saying, even if `b` is native, `a` should come first? (because it returns always `1` when either is native)

Reply:
It's actually unreal situation. We have only one native token in the list